arch/ua_network_tcp.c

```diff
@@ -525,20 +525,22 @@ UA_ServerNetworkLayer
 UA_ServerNetworkLayerTCP(UA_ConnectionConfig config, UA_UInt16 port, UA_Logger logger) {
     UA_ServerNetworkLayer nl;
     memset(&nl, 0, sizeof(UA_ServerNetworkLayer));
+    nl.deleteMembers = ServerNetworkLayerTCP_deleteMembers;
```

Comment:
does this "belong" into this series or is this something to break out into a PR of its own?
This is a fix to an overall(?) issues found by oss-fuzz?